app/Models/Visits/VaccinationSlotModel.cs

```diff
@@ -9,7 +9,8 @@ public class VaccinationSlotModel : BaseModel
         public DateTime Date { get; set; }
         public bool Reserved { get; set; }
         public virtual DoctorModel Doctor { get; set; }
-
+        public virtual VaccinationModel Vaccination { get; set; }
```

Comment:
czy to nie inwaliduje api w pewnym miejscu? testy z postmana przechodzą wszystkie?

Reply:
Nope, niby sprawdziłem i wszędzie jak jest używany VaccinationSlotModel to tylko przez DTO